lib/apidef/version.go

```diff
@@ -0,0 +1,181 @@
+package apidef
+
+import (
+	"errors"
+	"fmt"
+	"github.com/TykTechnologies/tyk/apidef"
+	"github.com/TykTechnologies/tyk/common/option"
+	"net/url"
+)
+
+const (
```

Comment:
I am not sure but it seems that this package requires exposed mebeers to be covered with godocs.